tests/framework/installer/ceph_installer.go

```
@@ -119,6 +114,58 @@ func (h *CephInstaller) CreateCephOperator(namespace string) (err error) {
 	return nil
 }
 
+func (h *CephInstaller) startAdmissionController(namespace string) error {
+	err := h.k8shelper.CreateNamespace(namespace)
+	if err != nil {
+		if errors.IsAlreadyExists(err) {
+			logger.Warningf("Namespace %q already exists!!!", namespace)
+		} else {
+			return fmt.Errorf("failed to create namespace %q. %v", namespace, err)
+		}
+	}
+	currDir, err := os.Getwd()
+	if err != nil {
+		return fmt.Errorf("failed to find current working directory. %v", err)
+	}
+	rootPath, err := findRookRoot(currDir)
+	if err != nil {
+		return fmt.Errorf("failed to find rook root. %v", err)
+	}
+	userHome, err := os.UserHomeDir()
+	if err != nil {
+		fmt.Println("failed to find user home directory")
+	}
+	err = h.k8shelper.MakeContext().Executor.ExecuteCommandWithEnv([]string{fmt.Sprintf("NAMESPACE=%s", namespace), fmt.Sprintf("HOME=%s", userHome)}, "bash", fmt.Sprintf("%s/tests/scripts/deploy_admission_controller_test.sh", rootPath))
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func findRookRoot(workingDir string) (string, error) {
+	const folderToFind = "tests"
+	workingDirectory, err := os.Getwd()
+	if err != nil {
+		return "", fmt.Errorf("failed to find current working directory. %v", err)
+	}
+	parentPath := workingDirectory
+	userHome, err := os.UserHomeDir()
+	if err != nil {
+		fmt.Println("failed to find user home directory")
```

need to return the error here

---

Done !